qtawesome/iconic_font.py

```diff
@@ -234,7 +234,15 @@ def load_font(self, prefix, ttf_filename, charmap_filename, directory=None):
         def hook(obj):
             result = {}
             for key in obj:
-                result[key] = unichr(int(obj[key], 16))
+                try:
+                    result[key] = unichr(int(obj[key], 16))
+                except ValueError:
+                    if int(obj[key], 16) > 0xffff:
+                        # ignoring unsupported code in Python 2.7 32bit Windows
+                        # ValueError: unichr() arg not in range(0x10000) (narrow Python build)
+                        warnings.warn("Your Python version doesn't support character {0}:{1}".format(key, obj[key]))
```

Comment:
Please fit this line in 79 columns (that's the max number of columns we use in our projects).